sunpy/net/vso/table_response.py

```diff
@@ -55,18 +55,12 @@ class VSOQueryResponseTable(QueryResponseTable):
     size_column = 'Size'
 
     @classmethod
-    def from_zeep_response(cls, response, *, client, _sort=True):
+    def from_zeep_response(cls, response, *, client):
         """
         Construct a table response from the zeep response.
         """
-        # _sort is a hack to be able to convert from a legacy QueryResponse to
-        # a table response.
-        if _sort:
-            records = iter_sort_response(response)
-        else:
-            records = response
-
         data = []
+        records = iter_sort_response(response)
```

Comment:
I had hoped to remove this but doing this breaks everything, so everything is sorted regardless now.